lib/git/duo/author_collection.rb

```diff
@@ -12,7 +12,7 @@ def where opts = {}
         result += opts.
           select {|k, _| authors.sample.respond_to? k }.
           map {|k, v|
-            authors.select {|o| o.send(k) =~ /#{v}/i }
+            authors.sort_by(&:key).select {|o| o.send(k) =~ /#{v}/i }
```

Comment:
Thoughts on placing `sort_by` on it's own line to be consistent with the rest of the chaining in the method?